### PR TITLE
As the jwt token expires, remove it from local storage

### PR DESCRIPTION
**Pull request by garvitgupta13**
## Issue that this pull request solves

 Closes: #503 

### Brief description of what is fixed or changed
As the jwt token expires it call the logout function and the admin is logged out from the dashboard. For this I have used the jwt-decode package to decode the token.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Admin/Admin.jsx [hunk 1]
```diff
@@ -17,12 +17,33 @@ import { Manage } from "./Components/Setting/Manage";
...
 
 export const Admin = () => {
   const [tab, setTab] = useState(1);
   const dispatch = useDispatch();
+
+  useEffect(() => {
+    const token = localStorage.getItem("token");
+    try {
+      const { exp } = decode(token);
+      // console.log("Decoded token: ", decode(token));
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Admin.jsx:32]: Hey @garvitgupta13 Can we please these commented lines

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Just to add on, when we logout the user can we please show the toast message of `Please login again!` instead of `successfully logged out!`

**Comment by garvitgupta13** [comment]
Mam I have tried to add a toast message by adding a key "expired" to localstorage before I call the logout function in key expire function but I don't know why it has created a new login error. Now when I try to login I face a backend issue.
These are the changes I made to Login.jsx
image
`expired`:
image

Error I am facing:
image
I have not made any changes to any backend file.
Can you please guide me

**Comment by Kajol-Kumari** [comment]
Hey @garvitgupta13 that's not the issue with your code. Seems like someone has messed up with the database again. We will fix it soon and get back to you

**Comment by Kajol-Kumari** [comment]
> Hey @garvitgupta13 that's not the issue with your code. Seems like someone has messed up with the database again. We will fix it soon and get back to you

This has been fixed now. Please try now.

**Comment by garvitgupta13** [comment]
What should be the `severity` of toast message: error , info , success.
All changes are done, soon I will be pushing them

**Comment by garvitgupta13** [comment]
Mam, please add the necessary `gssoc21` labels

**Comment by Kajol-Kumari** [comment]
> What should be the `severity` of toast message: error , info , success.
> All changes are done, soon I will be pushing them

Please keep it `info`

**Review by Kajol-Kumari — APPROVED** [review]
Hey @garvitgupta13 Thanks for patiently incorporating all the suggested changes.
Looks good to go now :)

**Pull request merged**